soc/nxp/imxrt/rt11xx/Kconfig.soc

```diff
@@ -0,0 +1,99 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_IMXRT_11XX
+	bool
+	select SOC_FAMILY_IMXRT
+
+config SOC_SERIES
+	default "rt11xx" if SOC_SERIES_IMXRT_11XX
+
+config SOC_MIMXRT1166
+	bool
+	select SOC_SERIES_IMXRT_11XX
+
+config SOC_MIMXRT1176
+	bool
+	select SOC_SERIES_IMXRT_11XX
+
+config SOC_MIMXRT1166_CM4
+	bool
+	select SOC_MIMXRT1166
+
+config SOC_MIMXRT1166_CM7
+	bool
+	select SOC_MIMXRT1166
+
+config SOC_MIMXRT1176_CM4
+	bool
+	select SOC_MIMXRT1176
+
+config SOC_MIMXRT1176_CM7
+	bool
+	select SOC_MIMXRT1176
+
+config MCUX_CORE_SUFFIX
+	default "_cm7" if SOC_MIMXRT1176_CM7 || SOC_MIMXRT1166_CM7
+	default "_cm4" if SOC_MIMXRT1176_CM4 || SOC_MIMXRT1166_CM4
```

Comment:
Asked in the main PR about this

Reply:
whats the "main PR"?

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/69368